mars/dataframe/execution/arithmetic.py

```diff
@@ -108,14 +109,22 @@ def _index_align_reduce(ctx, chunk):
 
 
 def _add(ctx, chunk):
-    inputs = [ctx[c.key] for c in chunk.inputs]
-    left = inputs[0]
-    if len(inputs) == 1:
-        right = chunk.op.scalar_value
-    else:
-        right = inputs[1]
-    ctx[chunk.key] = left.add(right, axis=chunk.op.axis,
-                              level=chunk.op.level, fill_value=chunk.op.fill_value)
+    try:
+        left = ctx[chunk.op.lhs.key]
+    except AttributeError:
+        left = chunk.op.lhs
+
+    try:
+        right = ctx[chunk.op.rhs.key]
+    except AttributeError:
+        right = chunk.op.rhs
+        # if right is scalar
+        ctx[chunk.key] = pd.DataFrame.add(left, right, axis=chunk.op.axis,
```

Comment:
Duplicate line with 126.

Reply:
line 26 has swap the location of `left` and `right`. Because pd.DataFrame.add() must have the first parameter be the dataframe.